install/linux/docker-ce/debian.md

```diff
@@ -137,7 +137,7 @@ from the repository.
 3.  Add Docker's official GPG key:
 
     ```bash
-    $ curl -fsSL {{ download-url-base}}/gpg | sudo apt-key add -
+    $ wget {{ download-url-base}}/gpg | sudo apt-key add gpg
```

Comment:
I think we standardize on using curl, also, the `-fsSL` options make sure that only a TLS connection is used (not sure if `wget` does the same by default);

What do you see if you only attempt to download the gpg file with curl? Does it produce some error? Could it be something in your environment (e.g. a corporate proxy)?

Reply:
I see no problem with the existing command: 

```
$ curl -fsSL https://download.docker.com/linux/debian/gpg | sudo apt-key add -
OK

$ lsb_release -d
Description:	Debian GNU/Linux 9.4 (stretch)
```

Reply:
@thaJeztah , I'm guessing that @alpmoi is behind a proxy.

Reply:
If the proxy environment is properly setup (`HTTP_PROXY` etc environment variables), those _should be_ automatically selected by `curl`. If there's a MITM proxy, perhaps it's failing because the certificate is invalidated by curl (if things are not properly setup)